modules/virt-configuring-a-live-migration-policy.adoc

```diff
@@ -0,0 +1,54 @@
+// Module included in the following assemblies:
+//
+// * virt/live_migration/virt-configuring-live-migration-policies.adoc
+
+
+:_content-type: PROCEDURE
+[id="virt-configuring-a-live-migration-policy_{context}"]
+= Configuring a live migration policy
+
+Use the `MigrationPolicy` custom resource definition (CRD) to define migration policies for one or more groups of selected virtual machine instances (VMIs).
+
+You can specify groups of VMIs by using any combination of the following:
+
+* Virtual machine (VMI) labels such as `size`, `os`, `gpu`, and other VMI labels.
+* Namespace labels such as `priority`, `bandwidth`, `hpc-workload`, and other namespace labels.
+
+All of the labels on the group of VMIs must match all of the labels in the policy for the policy to apply to the group of VMIs.
+
+[NOTE]
+====
+If multiple live migration policies apply to a VMI, the policy with the highest number of matching labels will take precedence. If multiple policies meet this criteria, the policies will be sorted by lexicographic order of the matching labels keys, and the first one in order will take precedence.
+====
+
+.Procedure
+
+. Create a `MigrationPolicy` CRD for your specified group of VMIs. For example, the following YAML configures a group with the labels `hpc-workloads:true` and `workload-type: db` :
```

Comment:
This sentence sounds not correct for me. In your MigrationPolicy example you have 2 Node and 2 VMI selector:

```
    namespaceSelector:
      matchLabels:
        hpc-workloads: true
        xyz-workloads-type: ""
    virtualMachineInstanceSelector:
      matchLabels:
        workload-type: db
        operating-system: ""
```

 and if I remember correctly the policy will be applied only when all selectors are met